doc/source/contributing/quest-available-datasets.rst

```diff
@@ -17,6 +17,11 @@ For interested readers, a preprint outlining the QUEST module and its applicatio
 
 `Argo Workflow Example <https://icepyx.readthedocs.io/en/latest/example_notebooks/QUEST_argo_data_access.html>`_
 
+QUEST uses the Argovis API to access Argo data, so users are encouraged to use the following citation:
+Tucker, T., D. Giglio, M. Scanderbeg, and S.S.P. Shen: Argovis: A Web Applications for Fast Delivery, Visualization, and Analysis of Argo data. J. Atmos. Oceanic Technol., 37, 401-416, https://doi.org/10.1175/JTECH-D-19-0041.1
+
+Citations for individual Argo datasets may be found at this link: <https://argovis.colorado.edu/about>
```

Comment:
```suggestion
Citations for individual Argo datasets may be found at this link: https://argovis.colorado.edu/about
```

Since the link is also the text, it renders showing the "<>".